spec/trace.py

```diff
@@ -0,0 +1,110 @@
+"""
```

Comment:
This folder is only user in parametric test? If yes, could you move it in `.parametric/` ?